buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidLibraryRuleComposer.groovy

```diff
@@ -46,6 +49,22 @@ final class AndroidLibraryRuleComposer extends AndroidBuckRuleComposer {
             testTargets.add(":${test(target)}")
         }
 
+        List<String> extraKotlincArgs = []
+        if (target.hasKotlinAndroidExtensions) {
```

Comment:
We definitely can, kapt is a plugin to the compiler just how kotlin android extensions is. We'll of course need the changes to buck that run each step involved when doing kapt.